app/views/account/tenants/edit.html.slim

```diff
@@ -1,4 +1,6 @@
 h1.col-md-6.col-md-offset-3 Edit Tenant
 .row
   .col-md-3.col-md-offset-3
-  	= render 'form', button_text: "Save Tenant", form_path: account_tenant_path
\ No newline at end of file
+    = simple_form_for [:account, @tenant],
```

Comment:
Why here? the difference only in button_text
move form to partial and just pass button_text